File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/TEST-2/zig/src/project_test.zig
const std = @import("std");
const mod_a = @import("mod_a.zig");
const mod_b = @import("mod_b.zig");

test "scale" {
    try std.testing.expect(mod_a.scale(2) == 6);
}

test "scale_and_offset" {
    try std.testing.expect(mod_b.scaleAndOffset(2) == 7);
}

File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/TEST-2/go/project_test.go
package test2go

import (
	"testing"

	"test2go/moda"
	"test2go/modb"
)

func TestScale(t *testing.T) {
	if moda.Scale(2) != 6 {
		t.Errorf("Scale(2) != 6")
	}
}

func TestScaleAndOffset(t *testing.T) {
	if modb.ScaleAndOffset(2) != 7 {
		t.Errorf("ScaleAndOffset(2) != 7")
	}
}
